topics/transcriptomics/tutorials/scrna-seq-basic-pipeline/tutorial.md

```diff
@@ -0,0 +1,910 @@
+> >
+> {: .solution}
+>
+{: .question}
+
+> ### {% icon question %} Question - Clustering resolution
+>
+> Do you think the clustering is appropriate? i.e. are there single clusters that you think should be separate, and multiple clusters that could be combined?
+>
+> ![Itm2a Expression](../../images/wab-umap-itm2a.png "Itm2a across clusters")
+>
+> > ### {% icon solution %} Solution
+> >
+> > Important to note, lest all bioinformaticians combine forces to attack the biologists: just because a cluster doesn't look like a cluster by eye is NOT enough to say it's not a cluster! But looking at the biology here, we struggled to find marker genes to distinguish the DP population, which we know is also affected by depth of sequencing. That's a reasonable argument that DP-M1, DP-M2, and DP-M3 might not be all that different. Maybe we need more depth of sequencing across all the DP cells, or to compare these explicitly to each other (consider variations on FindMarkers!). However, DP-L is both seemingly leaving the DP cluster and also has fewer knockout cells, so we might go and look at what DP-L is expressing in the marker genes. If we look at T-mat further, we can see that its marker gene - Itm2a - is only expressed in the top half of the cluster. You might consider sub-clustering this to investigate further, either through changing the resolution or through analysing this cluster alone.
+> > If we look at the differences between genotypes alone (so the pseudo-bulk), we can see that most of the genes in that list are actually ribosomal. This might be a housekeeping background, this might be cell cycle related, this might be biological, or all three. You might consider investigating the cycling status of the cells, or even regressing this out (which is what the authors did).
+> {: .solution}
+>
+{: .question}
+
+Ultimately, there are quite a lot ways to analyse the data, both within the confines of this tutorial (the many parameters that could be changed throughout) and outside of it (batch correction, sub-clustering, cell-cycle scoring, trajectories, etc.) Most analyses will still yield the same general output, though: there are fewer knockout cells in the mature T-cell population.
+
+{% icon congratulations %} Congratulations! You have interpreted your plots in several important ways!
+
+# Interactive visualisations
+
+Before we leave you to explore the unknown, you might have noticed that the above interpretations are only a few of the possible options. Plus you might have had fun trying to figure out which sample is which genotype is which sex and flicking back and forth between plots repeatedly. Figuring out which plots will be your *final publishable* plots takes a lot of time and testing. Luckily, there is a helpful interactive viewer export tool {% cite Moreno2020.04.08.032698 %} that can help you explore without having to produce new plots over and over!
+
+> ### {% icon hands_on %} Hands-on: Cellxgene
```

Comment:
Nice reference to CellXGene here

Reply:
There's no actual cellxgene publication! I put in one that references it... (by its GitHub site)